### ISSUE TITLE
Feature Request: Remote Storage for Savegames

### ISSUE THREAD

**Issue by rygos** [EasyRPG/Player#1213]
Allow to specify a remote storage for Savegames.
The EasyRPG Player could then read through a JSON API with a server corresponding Savegames.
For example, the web player already uses such a system. For example, you could access the Savegames anywhere without having to download them manually and upload them later.
The problem at this point is that the player must also be able to recognize the correct game and its version. For that I wrote another ticket.

**Comment by Ghabry** [comment]
Indirectly depends on #1085 because that could be part of a custom FS provided by the VFS layer. Like we alread do it in emscripten.

**Comment by Ghabry** [comment]
How do you plan to do the authentification? Putting an API-Token in the JSON?

For the savegame list the current emscripten solution used on rmarchiv (dictionary with keys 1-15 and value base64(lsd) could be used.

**Comment by rygos** [comment]
Yes, that could be used already.
Note: I am talking about the offline player. Not the WebPlayer.

Yes, I would use the JSON structure of the web server for the data transfer, makes the most sense. For authentication I would suggest to use jwt (JSON Web token).
